server/templates/pages/src/index.html

```diff
@@ -1,9 +1,6 @@
 <!doctype html>
-<!--[if lt IE 7]>      <html class="no-js lt-ie10 lt-ie9 lt-ie8 lt-ie7"> <![endif]-->
-<!--[if IE 7]>         <html class="no-js lt-ie10 lt-ie9 lt-ie8"> <![endif]-->
-<!--[if IE 8]>         <html class="lt-ie10 lt-ie9"> <![endif]-->
-<!--[if IE 9]>         <html class="lt-ie10"> <![endif]-->
-<!--[if gt IE 9]><!--> <html dir="{{ lang_dir }}" lang="{{ lang }}"> <!--<![endif]-->
+<!--[if lt IE 10]>      <html class="lt-ie10"> <![endif]-->
```

Comment:
This line should have `dir` and `lang` attributes, the same as R3. We neglected this in the past.